api/swagger.yml

```diff
@@ -4294,9 +4294,9 @@ paths:
       operationId: createPresignMultipartUpload
       summary: Initiate a multipart upload
       description: |
+        This operation is enabled only for the S3 blockstore.
         Initiates a multipart upload and returns an upload ID with presigned URLs for each part (optional).
-        Part numbers starts with 1. Each part except the last one has minimum size depends on the underlying blockstore implementation.
-        For example working with S3 blockstore, minimum size is 5MB (excluding the last part).
+        Part numbers begin at 1, and each part, except for the last one, must have a minimum size of 5 MB.
```

Comment:
The omitted part is important - I wouldn't remove it

Reply:
Which part exactly? Since I stated at the beginning of the description that this is only relevant to S3, "For example working with S3 blockstore" is not really relevant

Reply:
The minimum size of 5MB is for AWS S3, other S3 compatible storages (for example minio) might enforce different size minimum hence the clarification